jwql/website/apps/jwql/bokeh_containers.py

```diff
@@ -227,7 +227,7 @@ def edb_monitor_tabs(instrument):
     script : str
         The JS script to render dark monitor plots
     """
-    html_file_list = file_list[instrument]
+    file_list[instrument]
```

Comment:
This has no effect, and I don't see file_list defined - maybe the whole function can be deleted?